lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -363,16 +363,14 @@ public LockResponse lockWithFullLockResponse(LockClient client, LockRequest requ
             if (locks.isEmpty() || ((request.getLockGroupBehavior() == LOCK_ALL_OR_NONE)
                     && (locks.size() < request.getLockDescriptors().size()))) {
                 if (log.isTraceEnabled()) {
-                    log.trace(".lock(" + client + ", " + request + ") returns null");
+                    log.trace(".lock({}, {}) returns null", client, request);
                 }
                 if (requestLogger.isDebugEnabled()) {
                     requestLogger.debug("Failed to acquire all locks for {} for requesting thread {} after {} ms",
                             request, request.getCreatingThreadName(), System.currentTimeMillis() - startTime);
                 }
                 if (requestLogger.isTraceEnabled()) {
-                    StringBuilder sb = new StringBuilder("Current holders of the first ").append(
-                            MAX_FAILED_LOCKS_TO_LOG).append(" of ").append(failedLocks.size()).append(
-                            " total failed locks were: [");
+                    StringBuilder sb = new StringBuilder();
```

Comment:
Can we keep the initial writes to the StringBuilder here.  Keeps the diff smaller and using StringBuilder.toString and log formatting seems a bit strange.